### ISSUE TITLE
Module File Wrong For Version 2

### ISSUE THREAD

**Issue by ardan-bkennedy** [mattn/go-sqlite3#875]
Hi. Is the module file not specifying /v2 in the module name on purpose?

**Comment by aastein** [comment]
yeah, it's all messed up. search through existing open/closed issues. you need to manually set the version to to v1.14.0

**Comment by mattn** [comment]
Closing as addressed: the accidental v2.0.x releases are now retracted in go.mod, and stray v2 references were cleaned up.
